frontend/src/components/resourceMap/sources/definitions/relations.tsx

```diff
@@ -168,28 +183,48 @@ const pvcToPods = makeRelation(PersistentVolumeClaim, Pod, (pvc, pod) =>
 const podToOwner = makeOwnerRelation(Pod);
 const repliaceSetToOwner = makeOwnerRelation(ReplicaSet);
 
+const getCRToOwnerRelations = () => {
```

Comment:
only hooks can call other hooks so this should be have `use` prefix in the name to make sure it's clear that this is a hook